docs/helm-chart/parameters-ref.rst

```diff
@@ -66,6 +66,12 @@ The following tables lists the configurable parameters of the Airflow chart and
    * - ``airflowHome``
      - Location of airflow home directory
      - ``1``
+   * - ``airflowPodAnnotations``
+     - Extra annotations to apply to all Airflow pods.
+     - ``{}``
+   * - ``airflowConfigAnnotations``
+     - Extra annotations to apply to the main Airflow configs.
```

Comment:
```suggestion
   * - ``airflowConfigAnnotations``
     - Extra annotations to apply to the main Airflow configsmaps.
```

Reply:
I think `configmap` would be more appropriate than the plural here as these annotations don't get propagated out to other configmaps (`extra-configmaps`, `statsd-configmap`, and `webserver-configmap`). Thoughts?